backend/src/main/java/com/bakdata/conquery/models/worker/LocalNamespace.java

```diff
@@ -57,6 +73,22 @@ void registerColumnValuesInSearch(Set<Column> columns) {
 		}
 	}
 
+	@Override
+	String tryInnerResolveId(final EntityIdMap mapping, final EntityIdMap.ExternalId externalId) {
```

Comment:
dh, du startest jetzt eine Query pro Person ab? Das ist nicht so gut, weil da auch einfach ein paar Tausen leute drin sein dürfen.

Ich würde das eher so machen, dass du eine ganze Query (die gleiche wie regulär) abschickst und dann das Ergebnis mappest für die User an der Stelle des Endpunkts. Und in der Query eben den üblichen join machst.

Reply:
Meinst du so in etwa? 

```sql
with "cq_external" as (select '1' as "pid", '{[2012-01-01,2013-01-01)}'::datemultirange as "dates"
                       union all
                       select '2' as "pid", '{[2012-01-01,2013-01-01)}'::datemultirange as "dates"),
     -- ... more unions here)
     "cq_external_mapped" as (select "pid", "dates"
                              from "cq_external"
                                       inner join "persons" on "cq_external"."pid" = "persons"."pid")
select *
from "cq_external_mapped";
```

Ich weiß, dass wir darüber gesprochen hatten, aber ich hatte mich gefragt, ob das Vorgehen im Falle von IDs, die nicht aufgelöst werden können, so passt. Weil aktuell wird bei nicht erfolgreichem Auflösen die Query abgebrochen.

Reply:
ja sowas in die Richtung wäre besser.